internal/service/sagemaker/endpoint_configuration.go

```diff
@@ -448,6 +463,21 @@ func ResourceEndpointConfiguration() *schema.Resource {
 							Required: true,
 							ForceNew: true,
 						},
+						"routing_config": {
+							Type:     schema.TypeList,
+							Optional: true,
+							ForceNew: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"routing_strategy": {
+										Type:         schema.TypeString,
+										Required:     true,
+										ForceNew:     true,
+										ValidateFunc: validation.StringMatch(regexache.MustCompile(`^(LEAST_OUTSTANDING_REQUESTS|RANDOM)`), ""),
```

Comment:
Lets use string in list instead and check if API maybe has this as a list already (if not still better to check it as a static list)

Reply:
for example:

							ValidateFunc: validation.StringInSlice(sagemaker.ProductionVariantAcceleratorType_Values(), false),

Reply:
oh thanks! I was looking all over the validation package and missed this, exactly what I wanted.